metals/src/main/scala/scala/meta/internal/metals/SemanticTokenCapability.scala

```diff
@@ -0,0 +1,59 @@
+package scala.meta.internal.metals
```

Comment:
Can be moved to `mtags/src/main/scala/scala/meta/internal/pc` directory since it's only used in mtags.